upgrade-notes/backend_20240529_084428.md

```diff
@@ -1017,7 +1017,7 @@
     -   see #project-base diff to see changes you might need to apply in your tests
 -   `commerceguys/intl` has been updated to the latest version
     -   `IntlCurrencyRepository` and `NumberFormatterExtension` class methods have updated their interfaces to include strict types, you will need to update your usages of such methods in your project
-    -   `Phing` is no longer direct dependency, but is downloaded separately via `shopsys:phing:download` command. This is done directly in `composer.json` file, so you should follow changes in project-base to update `composer.json`, `phing` and other files accordingly
+    -   `Phing` is no longer direct dependency, but is downloaded separately via `php project-base/app/app/downloadPhing.php` command. This is done directly in `composer.json` file, so you should follow changes in project-base to update `composer.json`, `phing` and other files accordingly
```

Comment:
I think now, with the new approach, in the upgrading file, there should be your pull request only, am I right?